test-storybooks/portable-stories-kitchen-sink/nextjs/styles/globals.css

```
@@ -79,6 +79,9 @@
   margin: 0;
 }
 
+/* Custom variant for forcing hover */
```

```suggestion
/* Custom variant for forcing hover. Useful for Tailwind 4*/
```

I am suggesting a slight copy change on the comment so this doesn't get removed due to lack of context in a future theming update.

Thanks!